TypeCobol/Compiler/Diagnostics/CrossChecker.cs

```diff
@@ -755,6 +755,95 @@ public override bool Visit(DataDefinition dataDefinition)
             return true;
         }
 
+        class DataDefinitionChecker
+        {
+            public static void OnNode(DataDefinition dataDefinition, CommonDataDescriptionAndDataRedefines dataEntry = null)
```

Comment:
Done.
Side effects in LSR tests:

- Parent of level 88 requires a picture to avoid error
- Changes in completion results (for ex. parent of level 88 becomes numeric instead of alpha)